android/BOINC/app/src/main/java/edu/berkeley/boinc/BOINCActivity.java

```diff
@@ -271,8 +271,10 @@ private void dispatchNavBarOnClick(NavDrawerItem item, boolean init) {
 				fragmentChanges = true;
 				break;
 	    	case R.string.menu_help:
-	    		Intent i = new Intent(Intent.ACTION_VIEW, Uri.parse("http://boinc.berkeley.edu/wiki/BOINC_Help"));
-	    		startActivity(i);
+	    		startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("https://boinc.berkeley.edu/wiki/BOINC_Help")));
+	    		break;
+	    	case R.string.menu_bug_report:
+	    		startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("https://boinc.berkeley.edu/forum_forum.php?id=24")));
```

Comment:
Please use the same link as for the Manager, which just got [decided](https://github.com/BOINC/boinc/pull/2941#issuecomment-452515127) upon.

Reply:
Done.

Reply:
Done.